docs/docs/quickstarts/react.md

```diff
@@ -0,0 +1,117 @@
+
+![Create app in console](/img/angular/app-create-light.png)
+
+### Redirect URLs
+
+A redirect URL is a URL in your application where ZITADEL redirects the user after they have authenticated. Set your url to the domain the web app will be deployed to or use `http://localhost:3000/` because this will be the default of npm.
+
+> You should set dev mode to `true` as this will enable unsecure http for the moment.
+
+If you want to redirect the users back to a route on your application after they have logged out, add an optional redirect in the post redirectURI field.
+
+Continue and Create the application.
+
+### Client ID
+
+After successful app creation a popup will appear showing you your clientID.
+Copy your client ID as it will be needed in the next step.
+
+## React Setup
+
+### Create React app
+
+Create a new React app with the following command
+
+```bash
+npx create-react-app my-app
+```
+
+### Install oidc client
+
+You need to install an oauth / oidc client to connect with ZITADEL. Run the following command:
+
+```bash
+npm install oidc-react
+```
+
+This library helps integrating ZITADEL Authentication in your React Application.
+
+### Create and configure Auth Module
+
+With the installed oidc pakage you will need an AuthProvider which should contain the OIDC configuration.
+
+The oidc configuration should contain **openid**, **profile** and **email** as scope and **code** as responseType.
+In the code below the authority is already set to the issuer of zitadel.ch you can find this in the ZITADEL Console on you application.
+Replace the clientId value 'YOUr-CLIENT-ID' with the generated client id of you application in ZITADEL Console.
+
+
+```ts
+
+import React from 'react';
+import { AuthProvider } from 'oidc-react';
+import './App.css';
+const oidcConfig = {
+    onSignIn: async (response: any) => {
+        alert('You logged in :' + response.profile.given_name + ' ' + response.profile.family_name);
+        window.location.hash = '';
+    },
+    authority: 'https://issuer.zitadel.ch',
+    clientId:
+        'YOUR-CLIENT-ID',
+    responseType: 'code',
+    redirectUri: 'http://localhost:3000/',
+    scope: 'openid profile email'
+};
+
+function App() {
+    return (
+        <AuthProvider {...oidcConfig}>
+        <div className="App">
+        <header className="App-header">
+            <p>Hello World</p>
+    </header>
+    </div>
+    </AuthProvider>
+);
+}
+
+export default App;
+```
+
+### Run application
+
+Start your react application with the following command
+
+```bash
+npm start
+```
+
+Your browser should automatically open the app site or just go to `http://localhost:3000/`
+On opening the app in the browser you will be redirected to the login of zitadel.ch
+After successfully authenticating your user you will get back to you application.
+It should show a popup which says You logged in {FirstName} {LastName}
```

Comment:
i'd highlight the `You logged in` more either using `:` bold text or as code block